docs/mkdocs/javascript/shortcuts.js

```diff
@@ -0,0 +1,12 @@
+const isMac = window.navigator.userAgent.toLowerCase().includes("mac")
+const shortcut = isMac ? "⌘+k" : "ctrl+k";
+document.documentElement.style.setProperty('--md-search-extension-shortcut', shortcut);
+
+keyboard$.subscribe(key => {
+  const query = document.querySelector('[data-md-component="search-query"]')
+  if (key.mode === 'global' && key.type === 'k' && (key.ctrlKey || key.metaKey)) {
+    key.claim();
+    query.focus();
+    query.select();
+  }
+})
```

Comment:
It may be worth considering using the native keydown listener – this approach worked.
```
document.addEventListener('keydown', function (event) {
    if ((event.ctrlKey || event.metaKey) && event.key.toLowerCase() === 'k') {
        event.preventDefault();
        const query = document.querySelector('[data-md-component="search-query"]');
        if (query) {
            query.focus();
            query.select();
        }
    }
});

```